mopidy_emby/remote.py

```diff
@@ -66,9 +66,8 @@ def _password_data(self):
         """
         return {
             'username': self.username,
+            'pw': self.password.encode('utf-8'),
```

Comment:
could you explain why there is the need of the pw key?

Reply:
I too find it very messy to send the password as clear text.
But as the API description states. https://github.com/MediaBrowser/Emby/wiki/User-Authentication

> The password must be sent in the body, in three different form fields:

    pw - password in plain text
    password - password in Sha1
    passwordMd5 - password in MD5

> **IMPORTANT** - The Emby login API is in a state of transition, and this is why three different forms of the password are required. Beginning April 1, 2018, only the "pw" param will be required. Until then, all three are needed in order to support both newer and older servers.

Reply:
ok this is really ugly... but if they need it that way....